test/controllers/api/v2/ostree_branches_controller_test.rb

```diff
@@ -55,5 +55,13 @@ def test_compare
       assert_response :success
       assert_template "katello/api/v2/ostree_branches/compare"
     end
+
+    def test_branch_order
+      branch2 = @repo.ostree_branches.create!(:name => "def456", :uuid => "456uvw", :version => "1.0")
+      
+      get :index
+      expected_uuid = JSON.parse(response.body)['results'][0]['uuid']
+      assert_equal "123xyz", expected_uuid
```

Comment:
I may have used some incorrect wording in a previous comment. The second argument to a minitest assertion is the *actual* value and not the *expected* value http://ruby-doc.org/stdlib-2.0.0/libdoc/minitest/rdoc/MiniTest/Assertions.html#method-i-assert_equal

so, expected_uiud should be called 'uuid', 'actual_uuid' or something like that